include/sys/zfs_znode.h

```diff
@@ -38,7 +39,24 @@
 #include <sys/zfs_sa.h>
 #include <sys/zfs_stat.h>
 #include <sys/zfs_rlock.h>
+
+#else
+
+#include <sys/zfs_rlock.h>
+#include <sys/zfs_context.h>
+#include <sys/refcount.h>
+
+/* User mode znode emulation for ztest */
+#define ZTOZSB(zp) (zp)			/* unused */
+typedef struct znode {
+	uint64_t z_size;		/* unused, zeroed out */
+	uint64_t z_blksz;		/* unused, zeroed out */
+	uint64_t z_max_blksz;		/* unused, zeroed out */
+	zfs_rlock_t z_range_lock; 
```

Comment:
Yes, that's what I meant. Not need to define something if you don't really need it.